packages/affiliates/libs/processors.js

```diff
@@ -81,6 +83,12 @@ function EmpBalances(handlers = {}, { collateral, tokens } = {}) {
   collateral = collateral || Balances({ allowNegative: true });
   tokens = tokens || Balances({ allowNegative: true });
 
+  // Doesnt quite fit under umbrella of "balances" but this is the easiest place to set an expired flag.
+  let expired = false;
+  function isExpired() {
+    return expired;
+  }
```

Comment:
that is a good idea. it would also require removing the settleExpired events to keep balances from changing post expiry. After some consideration i think I will keep the expiry flag for this reason: it keeps the opinion of what to do with the balances up to the user of the balances class, rather than moving that opinion into the balances class by automatically setting them to 0 on expiry.